src/status_im/contexts/wallet/events.cljs

```diff
@@ -57,16 +57,16 @@
 (rf/reg-event-fx
  :wallet/get-accounts-success
  (fn [{:keys [db]} [accounts]]
-   (let [wallet-accounts     (filter #(not (:chat %)) accounts)
+   (let [wallet-accounts     (data-store/rpc->accounts accounts)
```

Comment:
The filter is already in the `rpc->accounts` method. Just removed the duplication.